### PR TITLE
Doc CockroachDB sink

### LINKED ISSUE #1512: Document: test: add a demo for cockroach sink
This issue tracks the documentation update needed for the merged PR #13308.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/13308
Source PR Merged At: 2023-11-14T05:30:46Z

### PR DESCRIPTION
**Pull request by emile-00**
## Info

- **Description**

  - Doc CockroachDB sink

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - https://github.com/risingwavelabs/risingwave/pull/13308

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1512

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Comment by emile-00** [comment]
@neverchanje Is there a data type mapping table worth including? From the sample SQL queries the type mapping seems to be 1 to 1.

**Review by neverchanje — APPROVED** [review]
Well done!

**Comment by neverchanje** [comment]
Sorry for my last review. I went through the content but i didnt see that you duplicated the pg sink doc to cockroachdb. It's unnecessary as cockroachdb is PG-compatible as well. You just need to link the cockroach guide to the pg guide, like how we wrote the redpanda doc

File: docs/guides/sink-to-cockroach.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- Ensure you already have a CockroachDB table that you can access and sink data to.
+  For additional guidance on setting up CockroachDB, refer to this [quick start guide](https://www.cockroachlabs.com/docs/cockroachcloud/quickstart).
+
+- Ensure you have an upstream materialized view or source that you can sink data from.
+
+- If you are running RisingWave locally from binaries and intend to use the native CDC source connectors or the JDBC sink connector, make sure you have [JDK 11](https://openjdk.org/projects/jdk/11/) or a later version installed in your environment.
+
+## Syntax and Parameters
+
+Since CockroachDB is PostgreSQL-compatible, the syntax and parameters are the same as creating a PostgreSQL sink. For the syntax and parameters, see [Create a Sink](/guides/sink-to-postgres.md#create-a-sink​) under Sink to PostgreSQL.
```
Comment (hengm3467) [inline docs/guides/sink-to-cockroach.md]: "under Sink to PostgreSQL" is unnecessary as we already provide the link.

**Pull request merged**